Sources/Apollo/GraphQLSubscriptionWatcher.swift

```diff
@@ -0,0 +1,34 @@
+import Dispatch
+
+public final class GraphQLSubscriptionWatcher<Subscription: GraphQLSubscription>: Cancellable
```

Comment:
What is the function of `GraphQLSubscriptionWatcher`? Couldn't `subscribe` be `_subscribe` and return the result of `send` directly?

Reply:
GraphQLSubscriptionWatcher started out based on the query watcher. 
Right now the only value add compared just returning the Cancellable is the "unsubscribe" function that may be more intuitive than "cancel" for terminating an active subscription. Apart from that it can be removed